imod/tests/test_msw/test_sprinkling.py

```diff
@@ -269,3 +270,40 @@ def test_simple_model_1_subunit(fixed_format_parser):
     )
     assert_equal(results["layer"], np.array([3, 2]))
     assert_equal(results["svat_groundwater"], np.array([1, 2]))
+
+
+def test_sprinkling_from_imod5_data__points(cap_data_sprinkling_points):
+    with pytest.raises(NotImplementedError):
+        msw.Sprinkling.from_imod5_data(cap_data_sprinkling_points)
+
+
+def test_sprinkling_from_imod5_data__grid(cap_data_sprinkling_grid):
+    # Arrange
+    # fmt: off
+    np.nan = nan
```

Comment:
Why are you ding this?

Reply:
That indeed is an error, I think I wanted to do nan = np.nan, to nicely outline the matrices.

Reply:
Fixed